# Generated by Django 4.0.3 on 2022-06-01 14:03

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('realestate_app', '0014_county_house_price_total'),
        ('realestate_occupancy', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='OccupancyEstimate',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('occupancy_estimate_value', models.FloatField(blank=True, null=True)),
                ('county', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='county_est_occ', to='realestate_app.county')),
                ('occupancy', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='occupancy_estimate', to='realestate_occupancy.occupancy')),
            ],
            options={
                'verbose_name': 'Occupancy_Estimate',
                'unique_together': {('occupancy', 'county')},
            },
        ),
        migrations.CreateModel(
            name='OccupancyError',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('occupancy_error_value', models.FloatField(blank=True, null=True)),
                ('county', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='county_err_occ', to='realestate_app.county')),
                ('occupancy', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='occupancy_error', to='realestate_occupancy.occupancy')),
            ],
            options={
                'verbose_name': 'Occupancy_Error',
                'unique_together': {('occupancy', 'county')},
            },
        ),
    ]
